org.adempiere.base.process/src/org/idempiere/process/ImportCSVProcess.java

```diff
@@ -150,10 +150,8 @@ protected void importFile(String filePath, IGridTabImporter csvImporter, GridTab
 		File outFile = csvImporter.fileImport(activeTab, childTabs, m_file_istream, Charset.forName(m_importTemplate.getCharacterSet()), p_ImportMode, processUI);
 		// TODO: Potential improvement - traverse the outFile and call addLog with the results
 
-		if (processUI != null)
-			processUI.download(outFile);
-		else if( getProcessInfo() != null ){
-			ProcessInfo m_pi = getProcessInfo();
+		ProcessInfo m_pi = getProcessInfo();
```

Comment:
Now this process is not showing the user the output file to download when in foreground